src/printer/multiline-array-printer.ts

```diff
@@ -14,13 +15,13 @@ function wrapInOriginalPrinterCall<T extends string = string>(
         const originalPrinter = getOriginalPrinter();
 
         if (property === 'print') {
-            const path = args[0] as AstPath;
             const options = args[1] as ParserOptions;
+            debugger;
+            const argsToUse = args[3] ? args : args.slice(0,3);
             const originalOutput = originalPrinter.print.call(
```

Comment:
This call is not working anymore, for an unknown reason

Reply:
The code that's throwing the error seems to be this:

https://github.com/prettier/prettier/blob/c0d464f3cfb56a0280d06941fdfff266f0ac89f7/src/language-js/print/angular.js#L14-L19

For some reason `node` in `args[0]` is indeed `undefined`. I'm not quite sure what `args[0]` actually is, it seems to be an instance of `AstPath`, but perhaps this can give some clue as to what might be the issue? I'll dig a bit more.